apps/sms/test/marionette/mocks/mock_split_view_mode.js

```diff
@@ -0,0 +1,14 @@
+/* global Services */
+'use strict';
+
+Services.obs.addObserver(function(document) {
+  if (!document || !document.location ||
+      document.location.origin !== 'app://sms.gaiamobile.org') {
+    return;
+  }
+
+  // Switch to split-view mode Inbox instead.
+  if(document.location.pathname === '/index.html') {
```

Comment:
IMO, this hack looks simpler and nicer than a manifest injection :)

I couldn't find it in the w3c specs: `location.replace()` is a synchronous function, right?